modules/stb/0.0.0-20241109-5c20573/overlay/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "stb",
+    hdrs = glob([
+        "*.h",
```

Comment:
Since there are a lot of libraries and they seem pretty unrelated, what do you think of adding one target per library? You can automate this with a list comprehension over `glob(["stb_*.h"])`.